website/app-templates/smarty/plugins/modifier.cachelink.php

```diff
@@ -11,7 +11,11 @@
  * Purpose:  outputs a cache link
  * -------------------------------------------------------------
  */
-function smarty_modifier_cachelink(\GeoKrety\Model\Move $move) {
+function smarty_modifier_cachelink($move) {
+    if ($move == null || !$move instanceof\GeoKrety\Model\Move) {
```

Comment:
Why not try `smarty_modifier_cachelink(?\GeoKrety\Model\Move $move` or `smarty_modifier_cachelink(\GeoKrety\Model\Move $move = null` ;)

Reply:
thoses 2 ways results in 

    Argument 1 passed to smarty_modifier_cachelink() must be an instance of GeoKrety\Model\Move or null, bool given

Reply:
> Then error in parent template, null should arrive there, no?

Yep, I updated parent template to not show empty list entries => `(bool(false))` in fact..

So I will revert instanceof ;)